File: src/main/java/eu/qwsome/sql/InnerTableJoin.java
package eu.qwsome.sql;

import eu.qwsome.sql.condition.Condition;

/**
 * Crate for join attributes.
 *
 * @author Lukáš Kvídera
 */
class InnerTableJoin extends TableJoin {

  InnerTableJoin(final String joinTable, final Condition condition) {
    super(joinTable, condition);
  }

  @Override
  CharSequence getPrefix() {
    return "";
  }
}

File: src/main/kotlin/eu/qwsome/sql/InnerTableJoin.kt
/*
 * Crate for join attributes.
 *
 * @author Lukáš Kvídera
 */
package eu.qwsome.sql

import eu.qwsome.sql.condition.Condition

internal class InnerTableJoin(
    joinTable: String,
    condition: Condition,
) : TableJoin(joinTable, condition) {

    override fun getPrefix(): CharSequence = ""
}
